src/AsyncGenerator.yml

```diff
@@ -3,6 +3,8 @@
   targetFramework: net461
```

Comment:
What this setting is about? Shouldn't it also be changed to .net core? If yes can it still be netcoreapp2.0?

I've noticed that compilation and debugging under .net core is a little bit faster - so my main desire is to avoid net461 compilation altogether in development process. So with .net core migration I'm hoping to get rid of the following AsyncGenerator exception:
>  [exec] Msbuild failed when processing the file 'E:\Git\nhibernate-core\src\NHibernate\NHibernate.csproj' with message: C:\Program Files\dotnet\sdk\2.1.602\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets: (198, 5): Assets file 'E:\Roman\Git\nhibernate-core\src\NHibernate\obj\project.assets.json' doesn't have a target for '.NETFramework,Version=v4.6.1'. Ensure that restore has run and that you have included 'net461' in the TargetFrameworks for your project.

So is this setting related to this error?

Reply:
> What this setting is about?  If yes can it still be netcoreapp2.0?

This setting set the target framework which is used to compile the project. When having multiple target frameworks it is required to set one for which the compilation will be done.

> Shouldn't it also be changed to .net core?

I forgot to change it, yes it should be changed to .net core.

>  If yes can it still be netcoreapp2.0?

It has to be one of the frameworks that the project has in the `<TargetFrameworks>` tag, so yes it can be `netcoreapp2.0`.

> So is this setting related to this error?

Yes it is related, you probably set in `NHibernate.dev.props` to only `netcoreapp2.0`.